3p/facebook.js

```diff
@@ -36,31 +36,65 @@ function getFacebookSdk(global, cb) {
 
 /**
  * @param {!Window} global
- * @param {!Object} data
+ * @param {!Object} data The element data
+ * @return {!Element} div
  */
-export function facebook(global, data) {
+function getPostContainer(global, data) {
+  const container = global.document.createElement('div');
   const embedAs = data.embedAs || 'post';
   user().assert(['post', 'video'].indexOf(embedAs) !== -1,
       'Attribute data-embed-as  for <amp-facebook> value is wrong, should be' +
       ' "post" or "video" was: %s', embedAs);
-  const fbPost = global.document.createElement('div');
-  fbPost.className = 'fb-' + embedAs;
-  fbPost.setAttribute('data-href', data.href);
-  global.document.getElementById('c').appendChild(fbPost);
+  container.className = 'fb-' + embedAs;
+  container.setAttribute('data-href', data.href);
+
+  return container;
+}
+
+/**
+ * @param {!Window} global
```

Comment:
Could you please make a reference to FB API, e.g https://developers.facebook.com/docs/plugins/comments